src/Crypto/Lol/Cyclotomic/Tensor/CTensor.hs

```diff
@@ -283,7 +286,12 @@ instance (TElt CT (ZqBasic q i), Dispatch (ZqBasic q i)) => CRNS (ZqBasic q i) w
   divGWrapper f = f
   gaussWrapper = error "Cannot call gaussianDec for ZqBasic"
 
-instance (Storable a, Storable b, CRNS a, CRNS b, CRTrans a, CRTrans b) 
+instance (Storable a, Storable b, 
+          CRNS a, CRNS b, 
+          CRTrans a, CRTrans b, 
+          ZeroTestable a, ZeroTestable b, 
```

Comment:
The instance constraints are due to the recursive calls in that instance and the new constraints that had to go on each wrapper function. The wrapper functions needed new constraints because the functions I'm wrapping have new constraints due to the weakening of TElt. CTensor did get uglier, but I'm not particularly worried about CTensor...